synapse/models/infotech.py

```diff
@@ -385,6 +385,31 @@ def getModelDefs(self):
                 ('it:dev:regval', ('guid', {}), {
                     'doc': 'A Windows registry key/value pair.',
                 }),
+                ('it:dev:repotype', ('taxonomy', {}), {
```

Comment:
The naming scheme for these has been tweaked. Would now be `it:dev:repo:type:taxonomy` :)

Reply:
Ah, so the `it:dev:hardwaretype` is just an old name. Gotcha :+1: